roles/ansible-test/tasks/ansible_test_changed.yaml

```diff
@@ -4,7 +4,7 @@
     chdir: "{{ ansible_test_location }}"
     executable: /bin/bash
   shell: |
-    for i in $(git diff origin/main --name-only|egrep '^plugins/.*.py'|xargs -r basename -s .py); do
+    for i in $(git diff origin/main --name-only|egrep '^plugins/.*.py'|xargs -r basename -s .py) $(git diff origin/main --name-only|sed -n 's#tests/integration/targets/\(.*\)/.*#\1#p'); do
```

Comment:
drive by review:

Is there a reason to hard code 'origin/main' ?  Rather then using the branch checked out by zuul?

```
{{ zuul.branch }}
```

Reply:
Thanks for the suggestion.